plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDataSource.java

```diff
@@ -376,7 +376,7 @@ protected Connection openConnection(@NotNull DBRProgressMonitor monitor, @Nullab
             try {
                 // Patch URL with new database name
                 if (CommonUtils.isEmpty(conConfig.getUrl()) || !CommonUtils.isEmpty(conConfig.getHostName())) {
-                    conConfig.setDatabaseName(instance.getName());
+                    //conConfig.setDatabaseName(instance.getName());
```

Comment:
Let's not leave commented code laying around. If we're removing a line, let's remove the line.

Reply:
It has been removed, thanks for your review.

Reply:
👍